README.md

```diff
@@ -82,44 +95,44 @@ Use aws cli with --profile 772123451421:role/onelogin-test-s3/myuser@mycompany.c
 
 ## Configuration File
 
-The configuration file is an `.ini` file with each section referring to a 
-OneLogin AWS application which can be authenticated against. There is also a 
+The configuration file is an `.ini` file with each section referring to a
+OneLogin AWS application which can be authenticated against. There is also a
 special section called `[defaults]` which has values to be used as defaults in
 other sections.
 
 ### Directives
 
 - `base_uri` - One of either `https://api.us.onelogin.com/` or `https://api.eu.onelogin.com/`
   depending on your OneLogin account.
-- `subdomain` - The subdomain you authenticate against in OneLogin.  
+- `subdomain` - The subdomain you authenticate against in OneLogin.
```

Comment:
Why remove all of these line breaks? I think they improve formatting.

Reply:
I just hit auto format which removed trailing spaces.

They're not line breaks, they're spaces I think. Let me double check. Looking at the raw files for both:
 - https://raw.githubusercontent.com/physera/onelogin-aws-cli/master/README.md
 - https://raw.githubusercontent.com/drewsonne/onelogin-aws-cli/feature/force-password-reset/README.md

I can't see where the formatting changes.

Reply:
The character in the source code is two spaces. Which is a line break in Markdown.

Reply:
Using line numbers from master as reference.

So the white space between `... OneLogin.` on line 94 and the beginning of line 95 is:

`20 20 0A 20 20`

So that's:
 -  two spaces 0x20, which are the ones being removed.
 - a new line 0x0A, and;
 - then two more spaces which are the indent on line 95.

Is that what you're referring to?

![image](https://user-images.githubusercontent.com/233054/40611207-50c87648-626d-11e8-9d34-e68249f6d755.png)

Reply:
Correct, space character is 0x20. Two space characters at the end of the line inserts a line break. The fact that the next line starts with two spaces is not relevant.

Reply:
Ah I see, gotcha now. Yeah sure. I'll revert that.

Reply:
@thnee I didn't add it back in here (https://github.com/physera/onelogin-aws-cli/pull/117/files#diff-04c6e90faac2675aa89e2176d2eec7d8R123) as it shouldn't have one anyway as the next line is a new `li`, but those are back in now.